argocd/resource_argocd_application_test.go

```diff
@@ -1630,6 +1662,54 @@ resource "argocd_application" "ignore_differences_jqpe" {
 	`, name)
 }
 
+func testAccArgoCDApplicationIgnoreDiffManagedFieldsManagers(name string) string {
+	return fmt.Sprintf(`
+resource "argocd_application" "ignore_differences_managed_fields_managers" {
+  metadata {
+    name      = "%s"
+    namespace = "argocd"
+    labels = {
+      acceptance = "true"
+    }
+  }
+
+  spec {
+    source {
+      repo_url        = "https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami"
+      chart           = "redis"
+      target_revision = "16.9.11"
+    }
+
+    destination {
+      server    = "https://kubernetes.default.svc"
+      namespace = "default"
+    }
+    
+    ignore_difference {
+      group                   = "apps"
+      kind                    = "Deployment"
+	  json_pointers           = ["/spec/replicas"]
```

Comment:
``` suggestion
      json_pointers           = ["/spec/replicas"]
``` 

nit: mixed tabs/spaces (not your fault these tests are painful in this regard - spaces in *.go files :vomiting_face: )